### PR TITLE
Attempt to fix launch fail

### PR DESCRIPTION
**Pull request by pskowronek**
Attempt to fix launch fail #1182

It might be related to recent update where things are run in parallel for faster startup, or it may not, or it may have start to manifest itself under such condition.

I was unable to reproduce, even having multiple tabs (this is only when this may occur), however I've spotted that JTabbedPane#getSelectedIndex may return -1 if no tab is selected. This index is being used in the HideableTabbedPane#updateTab (via update event).

@ahadas - I think you know the mechanics of those tabs better, can you take a look?

### CODE REVIEW

**Comment by pskowronek** [comment]
@ahadas - let's merge this fix and release minor update to muC?

File: mucommander-core/src/main/java/com/mucommander/ui/tabs/HideableTabbedPane.java [hunk 1]
```diff
@@ -188,7 +188,10 @@ protected void addAndSelectTab(T tab) {
 	 * @param updater - object that will be used to update the tab
 	 */
 	protected void updateCurrentTab(TabUpdater<T> updater) {
-		tabsCollection.updateTab(getSelectedIndex(), updater);
+		var idx = getSelectedIndex();
+		if (idx >= 0) {
+			tabsCollection.updateTab(idx, updater);
+		}
```
Comment (ahadas) [inline mucommander-core/src/main/java/com/mucommander/ui/tabs/HideableTabbedPane.java:194]: I've managed to reproduce it by:
1. stopping both threads that initialize the folder panels in the constructor of FileTableTabs
2. let the thread of the right folder panel continue and stop it in LocationChanger#tryChangeCurrentFolderInternal, after `mainFrame.setNoEventsMode(true);` is called
3. let the thread of the left folder panel continue

the thread of the left folder panel would raise the same exception
the explanation is that in no-event-mode, the selected index of FileTableTabbedPane is not set (see FileTableTabbedPane#setSelectedIndex) and therefore we get -1 from getSelectedIndex here

this change would solve that issue but the selected index won't be set properly
I'd prefer to see how we can avoid this situation

Comment (ahadas) [reply mucommander-core/src/main/java/com/mucommander/ui/tabs/HideableTabbedPane.java:194]: posted an alternative solution in #1201

**Comment by pskowronek** [comment]
I'm closing this PR (using alternative fix by @ahadas )

**Pull request closed without merging**